features/basic_replication.py

```diff
@@ -0,0 +1,51 @@
+import psycopg2 as pg
+from time import sleep
+
+from lettuce import world, steps
+
+PATRONI_CONFIG = '{}.yml'
+
+
+@steps
+class BasicReplicationSteps(object):
+
+    def __init__(self, environ):
+        self.env = environ
+
+    def start_patroni(self, step, pg_name):
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/cyberdemn/patroni/file/2110318101/issues/source?bid=3127594&fileBranchId=3144623#l15)